tests/e2e/build_test.go

```diff
@@ -0,0 +1,95 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/fsouza/go-dockerclient"
+)
+
+// Kedge location
+var KedgeLoc = "$GOPATH/src/github.com/kedgeproject/kedge"
+var err error
+
+func runCmd(cmdS string) ([]byte, error) {
+	var cmd *exec.Cmd
+	var out, stdErr bytes.Buffer
+	cmd = exec.Command("/bin/sh", "-c", cmdS)
+
+	cmd.Stdout = &out
+	cmd.Stderr = &stdErr
+
+	err := cmd.Run()
+	if err != nil {
+		return nil, fmt.Errorf("error running command %v: %s", cmd, err)
+
+	}
+	return out.Bytes(), nil
+}
+
+func DeleteRegistry(t *testing.T) {
+
+	_, err = runCmd("docker stop kedge_registry")
+	if err != nil {
+		t.Error(err)
+	}
+
+	_, err = runCmd("docker rm -v kedge_registry")
+	if err != nil {
+		t.Error(err)
+	}
+
+	t.Log("The registry has been deleted successfully")
+}
+
+func TestBuild(t *testing.T) {
+	// Context Dir to be passed for kedge build
+	var contextDir = KedgeLoc + "/docs/examples/build"
+
+	endpoint := "unix:///var/run/docker.sock"
+	cli, err := docker.NewClient(endpoint)
+	if err != nil {
+		t.Error(err)
+	}
+
+	dockerRegStart := "docker run -d -p 5000:5000 --restart=always --name kedge_registry registry:2"
```

Comment:
You can also add the flag `--rm` which will delete the docker container once the registry is stopped!

Reply:
This will reduce the whole hassle of deleting the container

Reply:
:+1: